src/main.rs

```diff
@@ -494,6 +507,7 @@ async fn start_ldk() {
 		keys_manager.get_node_secret(),
 		&ephemeral_bytes,
 		logger.clone(),
+		Arc::new(IgnoringMessageHandler {}),
```

Comment:
Shouldn't you be able to use `new_routing_only` or `new_channel_only` to avoid this parameter?